chain/network-primitives/Cargo.toml

```diff
@@ -10,6 +10,7 @@ tokio = { version = "1.1", features = ["full"] }
 chrono = { version = "0.4.4", features = ["serde"] }
 borsh = "0.9"
 serde = { version = "1", features = [ "derive" ] }
+sha2 = "0.9.8"
```

Comment:
I think now that there's `borsh_hash` helper, changes to Cargo.toml are not needed?

Reply:
Thanks, fixed.